apps/meteor/app/oembed/server/server.ts

```diff
@@ -273,37 +306,10 @@ const rocketUrlParser = async function (message: IMessage): Promise<IMessage> {
 			continue;
 		}
 
-		if (!isURL(item.url)) {
-			continue;
-		}
-
-		const data = await getUrlMetaWithCache(item.url);
-
-		if (!data) {
-			continue;
-		}
-
-		if (isOEmbedUrlContentResult(data) && data.attachments) {
```

Comment:
Same as before, if we are removing this only and only because it's not being returned by `getUrlMetaWithCache `, it would be better to keep as is here, and create another PR, so it's easier to track the changes.

Reply:
[This](https://github.com/RocketChat/Rocket.Chat/blob/14fc38e60e029ddfd6c1732c80a649fd7014fdca/apps/meteor/app/oembed/server/server.ts#L124) is the correct link for the line I mentioned :nerd_face:
I was just not so sure about doing that when I created this PR since doing that (keeping this line) would require me to add another param (the `attachments` array) to the `parseUrl` function just to remove it later in another PR. Does it look fine to you?